spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe "Food Index Page", type: :feature do
+  it "displays list of attributes for food search results", :vcr do
+    food1 = Food.new(gtinupc: 124, description: "yummy food", brand_owner: "General Mills", ingredients: "hot dog, bun, mustard")
```

Comment:
Typically, it's more common to pass in the data as a hash as you would expect it to come in from the API data, so all those same fields wrapped in a  { } to indicate that you're passing in a hash that's been parsed from the JSON response